### PR TITLE
SOFT FORK: Today, BitClout joins the environmental movement.

### PR DESCRIPTION
**Pull request by diamondhands0**
I am proud to announce that the dev community has committed to moving to
a zero-waste consensus mechanism, and that it is taking a major step in
that direction with a new update.

Cryptocurrencies have gotten a lot of heat recently for being harmful to
the environment.

This is due in large part to "proof of work," a consensus mechanism that
burns CPU power to secure the network.

BitClout has relied on proof of work to get off the ground, but since
launching the dev community has already invested in consensus mechanisms
that allow BitClout to operate on orders of magnitude less power than
most existing proof of work cryptocurrencies.

Now, in a show of solidarity with the environmental movement, and to
take the first big step toward a zero-waste consensus mechanism,
BitClout is deploying a new update that reduces its power consumption by
a full order of magnitude.

BitClout is reducing the block reward, and thus the amount of
environmental waste its consensus mechanism incurs, by a factor of ten.

This was a difficult change to make, and it took the input of several
major node operators to deploy it safely.

We understand that miners in particular may not like this change;
however, there is now consensus around the decision to push the limits
of the BitClout chain toward conservation rather than allowing the
depletion of what are ultimately finite environmental resources.

There is still more work to do to move to a zero-waste consensus
mechanism without compromising on decentralization (e.g. full proof of
stake).

But the dev community is committed to getting there sooner rather than
later, and we hope this change shows just how committed we are.

### CODE REVIEW

**Review by lazynina — APPROVED** [review]

**Review by redpartyhat — APPROVED** [review]

**Comment by iPaulPro** [comment]
Is there any plan to have the **actual community** start to vote on some of these huge decisions? 

Or perhaps at least discuss it before force-pushing to main?

**Comment by dezzyboy** [comment]
is this really about the energy usage? with 2TH of network hashrate has no effect what so ever on the environment. not sure when mining will stop but would have been nice to get a heads up.

**Comment by diamondhands0** [comment]
@iPaulPro I've very sorry this feels like it was sprung on you all. That was not the intent, and it's meant to be a positive change that will significantly increase how seriously BitClout is taken by the broader crypto ecosystem in light of all of the controversy over environmental-friendliness. It does also significantly decrease BitClout's footprint, which is a good thing both for energy usage but also for the cost of running the network.

In the future, changes like these will be submitted as CIPs (Clout Improvement Proposals) similar to BIPs in Bitcoin. The only reason we did not go through that process for this change is because we have not yet set up that repository. That said, it will be set up shortly, and it will allow the devs to get a lot more input before merging changes like this.

**Comment by tijno** [comment]
@diamondhands0 how can we as a community help put CIPs in place? whats left to be done right now? If anything needs reviewing happy to contribute as I am keen to help and get this right.

**Comment by FreeTrade** [comment]
Have you given any thought to MEV - miner extractable value.

Seems like miners could extract a lot of value by re-ordering or front-running creator coin transactions.

**Comment by HPaulson** [comment]
Just to toss my two cents into the mix:

First and foremost, I think it's important to mention that this is a great move in the right direction. Over the past four years, we've seen countless examples of proof-of-work consensus mechanisms failing to scale and be effective for daily use -- cause them to be slow, and extremely expensive. Projects like Ethereum, which have been greatly impacted by such failures, are now quickly converting to a cheaper and faster consensus model: Proof of Stake.

For these reasons, I'm unsure why the development team would use proof-of-work in the first place. The model is old, proven to lack scaling capability, is very slow, and requires a ton of electricity. For a social media blockchain, these aspects are vital. It much be fast, cheap for users, and easy to secure due to the number of transactions used, and the fact that they're by users not always making monetary transactions -- unlike most chains. Although BitClout works just fine with proof of work as of now -- the current blockchain won't scale, will be way too expensive for users, and overall wouldn't allow BitClout to expand to mass adoption. For this, a move to proof of stake (as the comments in the PR hint to) would be extremely ideal.

Now as for the PR itself, I'm unsure why this exists. Although the mindset of moving to a energy-minimal, fast, and cheap blockchain is great, this short-term solution is quite horrible and doesn't make much sense. While the intentions are good, shrinking the block reward makes mining less profitable, therefore making the network less secure due to lack-of-incentives. I think the idea here is good, but this for sure isn't the solution. If we're sticking with this solution, though, a larger change such as proof of stake needs to be in the pipeline ASAP.

As for CIP's, I think this is a great idea, and a vital one. While the chain is already decentralized, and the community could easily revert this change through the use of running 51% of nodes, I think having a move formal method of improvements with community consensus would make a ton of sense.

**Comment by tijno** [comment]
>  For this, a move to proof of stake (as the comments in the PR hint to) would be extremely ideal.

Agree very much @HPaulson and I would welcome some clarity about timescales about this, especially considering current Bitclout ownership and impact on CIP influence/decisions.

And like you im looking forward to CIP process going live soon.

**Comment by HPaulson** [comment]
@tijno Agreed. Time scaling is very important. Another thing is like to be very very clear about, however, is this: The core dev team doesn't matter. They don't own BitClout, and don't want to. 

I see way too much of the community acting as if they have the final say on these decisions -- they don't, and that's a great thing!

CIP's will be a good organization method, but just keep in mind that the protocol is in the hands of the community, not the dev team.

**Comment by tijno** [comment]
> The core dev team doesn't matter. They don't own BitClout, and don't want to.

In theory you are right - anyone can now take the code and restart the chain at any point in time.

In practice right now though they do decide & implement key changes quickly like we have seen. I am not sure anyone else forking at this point would get traction - theres only about 2k passionate users, and 99% of those will stick with bitclout.

Not bashing it though as any project from inception to opensource will go through a handover phase like this, where the rains have to be handed over carefully and slowly. 

> CIPs ... which of course didn't occur

I would love to see it soon, but if any further major changes are in the pipeline for the next 2 weeks - it may make sense to hold off with formally adopting the process until there is a point of stability. Eg - if I had to choose between major exchange listing or CIP - id be happy to hold off on CIP for a few weeks.

**Comment by MateosDev** [comment]
Wouldn't lowering the mining incentive this quickly put the network at risk for a 51% attack due to so many simply pointing their rigs at another blockchain?

**Comment by CreativeTransmissions** [comment]
This is excellent news! With the climate crisis at the top of the agenda for most countries and the widely publicised environmental impact of cryptocurrency, this would have been a major blocker for many people to join the network.

Younger generations of creators are the most aware of the climate crisis. To get mass adoption this should be something to shout about otherwise the network growth will stall as the climate crisis worsens and awareness of it increases.

**Comment by HPaulson** [comment]
@tijno Of course yes. In practice the devs do most of the changes at their discretion, however in reality the community does control the entire protocol -- and I think it's very vital that we approach these changes with this mindset.

----

@MateosDev This is the worry -- yes. As incentives lower, the network becomes less secure, which is of course very dangerous

**Comment by JRedinger** [comment]
I fully support the move to POS, though I am concerned at the lack of opportunity for community consensus on this. 

With that said, this move is part of quite an aggressive supply reduction overall very early in BitClout's lifetime - before the majority of coins have had time to distribute naturally via the incentives provided. Fair distribution of coins is critical to reaching an ethical consensus in POS decentralization. I trust that the greatest holders of BitClout are content to yield the power they inherit as a result of this unilateral decision that is taking place without community involvement, or I fear a rift may start to form in the trust we have all built with each other. 

I continue to trust the hands at work here. The work and vision I see here is truly inspired and noble. But I implore the decision-makers behind BitClout to please consider the foundation of the future that they are crafting. Full decentralized trustlessness of BitClout has not yet been achieved, as evidenced by the fact that unilateral decisions can still be made - many of us have cast ourselves 'all in' on BitClout on the trust that has been built by the developers of the project over time. Please take care not to fracture our fragile community. Until development is truly decentralized, our trust is in you.

**Comment by tijno** [comment]
so well put @JRedinger 

> Fair distribution of coins is critical to reaching an ethical consensus in POS decentralization.

distribution is essential. staking (and thus voting) power should probably not rest with a few beneficiaries.

we cant predict what will happen but we know the signs that show we remain on track.

on track to achieve the vision that got us all so excited .

the team have earned a level of trust with me that todate they have delivered on everything they said they would.

despite all the "scam" outcries on twitter and socials.

one by one their objections got addressed until there were none left.

exchange listing / offramp is the remaining item on the "first 100 days in office" list i think.

i cant see the whole puzzle yet, but hope this was a key piece. 

and i hope the miners/pool operators that invested huge amount of time and money to get ready for mining did so in the knowledge that this change was maybe looming. 

cant wait to see what next week brings.

**Comment by connerdouglass** [comment]
This is all very frustrating, so I want the record to reflect that I am strongly against moving away from proof of work, especially when it's done in this manner.

If this upcoming proof of consensus change is merged **without first being reviewed and approved by the actual community** here on GitHub, I don't think it would be unreasonable for us to consider a community-controlled hard fork.

**Comment by dezzyboy** [comment]
> This is all very frustrating, so I want the record to reflect that I am strongly against moving away from proof of work, especially when it's done in this manner.
> 
> If this upcoming proof of consensus change is merged **without first being reviewed and approved by the actual community** here on GitHub, I don't think it would be unreasonable for us to consider a community-controlled hard fork.

i think a community-controlled hard fork is a little bit far fetched and definitely not needed and in fact could set back this amazing new innovation. pos is great and i support it. im just disappointed in the fact that no heads up was given. but with that being said i think let us give it a chance to blossom in terms of distribution of the premine. depending on how that goes could turn out to be a great move for all. only time will tell.

**Comment by tijno** [comment]
I just feel for korbin who i think worked 20 hour days for weeks to get his pool launched. Im sure he did well out of it but also invested lots to get it build & tested.

Then to be told 2 weeks after it gone up in smoke ...

I hope he had knowledge of this or will be made good.

Miners can just switch to the next coin. Probably back to ETH for now.

**Comment by HPaulson** [comment]
> This is all very frustrating, so I want the record to reflect that I am strongly against moving away from proof of work, especially when it's done in this manner.
> 
> 
> 
> If this upcoming proof of consensus change is merged **without first being reviewed and approved by the actual community** here on GitHub, I don't think it would be unreasonable for us to consider a community-controlled hard fork.
> 
> 

Proof of stake is a much, MUCH better consensus model. There's no reason not to support it -- unless you're just trying to capitalize on the community through mining.

Something as large as POS would likely be a CIP. If not, we always have the ability to hardfork, however keep in mind pos would be a **good change**, so no reason to do so. We should tread lightly in all changes though, as I said prior.

**Comment by dokluch** [comment]
> This is all very frustrating, so I want the record to reflect that I am strongly against moving away from proof of work, especially when it's done in this manner.
> 
> If this upcoming proof of consensus change is merged **without first being reviewed and approved by the actual community** here on GitHub, I don't think it would be unreasonable for us to consider a community-controlled hard fork.

I totally agree. With mining now 10 times less profitable and POS not implemented (and no solid plans on that), seems like BitClout is not going to stay in the current state for long. I do support POS models, but again, this all came out of the blue, as it seems.

**Comment by paulmp-git** [comment]
Mining isn't necessarily bad for the environment... my house has solar panels, so the electricity used is about as clean as we can get at the moment. It is really strange to see this just sprung on the community with no input or real time to evaluate it.

**Comment by HPaulson** [comment]
@paulmp-git I think the environmental factor is less of the issue at hand -- and more of a PR thing for BitClout. A LARGE majority of BTC is mined with clean energy, yet its still looked at as negative for the environment.

The larger factor at hand, in my opinion, is the speed & scalability of the network. POW isn't scalable. It's slow, expensive, and just lacks the ability to grow -- just take a look at eth gas fees over time. I think PoS is vital, just maybe not for the reasons they described... no matter how we get there, it's the end goal that matters

**Comment by connerdouglass** [comment]
> Proof of stake is a much, MUCH better consensus model. There's no reason not to support it -- unless you're just trying to capitalize on the community through mining.
> 
> Something as large as POS would likely be a CIP. If not, we always have the ability to hardfork, however keep in mind pos would be a **good change**, so no reason to do so. We should tread lightly in all changes though, as I said prior.

I have never done any mining at all, actually. My reason for being against POS it is that it reduces the decentralization of the network, particularly on a network that just underwent a two month pre-mine.

I'm totally open to being proven wrong. I just don't buy the "environmental impact" arguments against crypto mining. It's all a misguided attempt at marketing and mass appeal, I think.

**Comment by HPaulson** [comment]
@connerdouglass Yup -- I 110% agree that the environmental reason is all PR. However, mining is definitely an old consensus model that can be replaced. 

As far as decentralization:

It Doesn't compromise on security what-so-ever -- in fact, it does the opposite. The premine of BitClout is not entirely owned by the development team. Wallets held include large seed investors, a fund to pay for exchanges, and much more. The investor wallets, however, are the important ones. These wallets contain BitClout which the core team has access to, but are not allowed to control/stake without discretion of the investor. This means that the team would need 51% of the supply (roughly 5.5m coins) Yk stake and centralize the network. With 6m already purchased by the community, this would be extremely challenging.

**Comment by connerdouglass** [comment]
@HPaulson That's all fair, but just because it's "old" does not mean mining should be replaced. I prefer to think of mining as "tried and true". Lots of new things are worse than the old things!

**Comment by tijno** [comment]
@HPaulson you know this as a fact, or you are speculating ? 

ive not heard this explanation before as the reason for limiting supply ?

**Comment by HPaulson** [comment]
@tijno I know this as a fact. Not sure how much I'm allowed to say further on the subject

**Comment by tijno** [comment]
maybe dont say anything more in that case

**Comment by teambrixmusic** [comment]
MEV - miner extractable value is excellent and wish it were considered.

I feel terrible for the people who purchased expensive new equipment just to mine Bitclout.

POS has moral issues. First off, it lends to less decentralization. Secondly, often, the rich get richer and those with smaller stakes stay that way for the most part, despite their contributions. It emphasized money, not community (which is the underlying value of a "social" blockchain.) Projects primarily intended to be used for dApps like games and social features really need to consider everyone, not just the largest stakeholders. In this sense, I think dPOS (ie hive/steem) is a little "better" in as far as at least smaller stakers can "vote" on which nodes get the blocks. And that incentivizes nodes to compete for the approval of who they represent by providing value in return.

Just throwing my opinion in here.

**Comment by iPaulPro** [comment]
> This was a difficult change to make, and it took the input of several
major node operators to deploy it safely.

@diamondhands0 I'm very interested to hear about these "major nodes". Who are these operators?

You often speak about the "dev community". Who makes up this community? 

Looking forward to your response.

**Comment by jazzyjackson** [comment]
Re: 6 million coins being adequate distribution for PoStake, I’m only familiar with ETH but they require 32 ETH to participate, is this because the number of stakeholders should be kept low for technical / sync reasons?

I guess I’m close to re-hashing (lol) the whole debate on Staking vs Working, but I just want to jump on the bandwagon that sees this coin as very obviously not evenly distributed. Maybe non-investor normies hold the majority of coins, but if PoS has a minimum threshold, you can cut off the long tail of those accounts as they’re ineligible to secure the network.

**Comment by sull** [comment]
Ok @diamondhands0 I and others do expect an elaboration on this topic ASAP. So please prepare that with the team and release it to the community. It would be wise to do this soon than later. It is not that this was the wrong decision necessarily but there needs to be more explanation and respect.... and an apology would also be appropriate.

**Comment by nbardy** [comment]
The Environmental angle is a flimsy argument. Changes like this deserve well reasoned argument, not hand wavey PR.

I think the goal of moving to proof of stake is valid. But the smoke and mirrors here feels dishonest.

**Comment by sull** [comment]
> The Environmental angle is a flimsy argument. Changes like this deserve well reasoned argument, not hand wavey PR.
> 
> I think the goal of moving to proof of stake is valid. But the smoke and mirrors here feels dishonest.

It does feel like PR a la Elon influence on BTC. Of course we should always welcome power use optimizations and in most cases, not perpetually rely on PoW (that was never going to happen here). I expected a transition to Proof of Stake at some point so that part is no surprise and I welcome it. But this soft fork explanation does not fly as the entire story.

**Comment by diamondhands0** [comment]
Hey everyone. First off, I am again very sorry that this change was rushed through. Again, that was not the intent-- we are just, frankly, in a mad dash to a major announcement next week and trying to get all of our ducks in a row before then. On the bright side, I am happy to say that, after next week, it will be much harder to make changes this quickly and this unilateraly (I can't say more without giving away the announcement). This is also why the CIPs change is not only beneficial, but actually needed. And @maebeam will launch it shortly-- there is a private repo, she's just doing *a lot* right now.

I've also had the opportunity to read through all of the Discord channels and this entire thread, making notes as I go through to remember everything. I don't always do this, but I want you all to know that I did that in this case because I think it's really important to address everyone's opinion here.

First, who did we talk to about this change? Just to be very clear, we talked to Korbin at length about the change well in advance of making it, we talked to a couple of the top names on bithunt.com (don't want to drag them into this), and we notified @lobovkin (though I do wish we could have given him more time to discuss it than we did). We should have talked to you all as well, and going forward the CIPs repo will be the mechanism for this. But, again, because we are currently in a mad dash, we had to cut things a little short this time. Again, in the future it will be much more difficult to push changes this quickly but I do want to emphasize that even in this case there was consensus around this decision being a good one among the people we talked to.

Now, why is this change important? There are other reasons than just the environment:

- All of us hold BitClout. We are all aligned by the fact that we all own the same asset. Mining is fundamentally inflationary. It prints BitClout, which then must be dumped in order to cover the miner's costs. And miners don't dump because they're bad people, they dump because it's the only way to operate: If you earn 1 BitClout, but your cost to earn it was .95 BitClout, then you must dump .95 BitClout to cover that cost. At current prices, the emission rate would have resulted in ~$20M+ USD being dumped every year, which would have had a very negative effect on the ecosystem. And who would even benefit from that? Yes, the miners would get .05 BitClout for every 1 BitClout they mined, but the other .95 would basically go toward wasting energy and hardware. Put another way, every .05 you take away from mining gives .95 back to existing BitClout holders (that would otherwise have been spent on wasting electricity). It's a hard thing for people to understand this, which is why I didn't go into it in my main announcement, but this is how it works.
- Many now understand the negative effects of proof of work. In my opinion, the fact that the environment is damaged by this activity is sufficient to make it worthwhile to get rid of. But even if you don't care about that, it is still the case that we're getting to a point where major public figures and funds can't be associated with something that doesn't have a serious plan to move away from it. We started with proof of work because it is, in my opinion, the best way to launch a truly decentralized platform in a bottom-up fashion. But at some point, not moving away from it is going to hurt BitClout's adoption and its narrative in a way that is totally avoidable. We have the power to take a very aggressive stance against proof of work now that the platform is fully open and sufficiently decentralized, and I truly believe it is the best thing for the platform to do that, even if you don't care about the environment.

Lastly, I want to make a few one-off points about things I came across in case people are concerned about them:

- On MEV and 51% attacks: BitClout is resistant to these attacks at present due to the "trusted block producer" setup. I'm not going to go into it here, as this deserves a post on its own, but you can trace the `--trusted_block_producer` flag and read the comments if you're curious about how it works.
- There is some chatter about how people who purchased mining equipment are hurt by this change. That is not true because they can trivially pivot their equipment to another blockchain and make money that way. Unless someone actually spent time writing a mining pool, like Korbin or @lobovkin, all of their investment should be very easy to repurpose if they want.
- On the genesis block: Everyone in the genesis block purchased coins from the bonding curve, same as any user purchases BitClout today (before the Deflation Bomb). All BitClout in existence came from either: 1) a legitimate purchase from the bonding curve with Bitcoin as it's defined in supply.go and described by @tijno in one of his posts, 2) mining, or 3) the 2M BitClout founder reward, which is part of the genesis block. The reason the genesis block is huge is because there was a hard fork that compressed all the purchases/balances between ~November 2020 until ~March 2021. This is one of the things I most regret about our launch because it has caused a lot of confusion, but I'm setting the record straight here that no BitClout was given away for free outside of the 2M, and none of the other holders in the Genesis block got coins outside of the bonding curve.

I know the above will not address all concerns, but I do hope it helps shed some light on things. I won't be able to respond to everything, but if it's important I hope you all understand that I'll put the time in, and give me the benefit of the doubt if I don't.

Lastly, I do want to say that, although I am disappointed in myself for not being more thoughtful about how we deployed this change, I am absolutely and unbelievably thrilled that I am part of a community that cares as deeply about this project as you all do right now. The fact that so many people are this invested in the platform is a very positive thing, and as someone who's been working on the project for over two years now, with over one year of it totally alone, it moves me in a very profound way.

Thank you all again for being a part of this, for encouraging us to be better, and please know that we are 100% committed to improving all the time, together.

**Comment by tijno** [comment]
Thanks for posting this thorough update. Will a summary be posted on bitclout ?

**Comment by xAlisher** [comment]
> Thank you all again for being a part of this, for encouraging us to be better, and please know that we are 100% committed to improving all the time, together.

Thank you for spotlighting the issue.  Hopefully, all bold moves like this will be announced and discussed more widely (and posted to the BitClout, not only GitHub)

**Comment by automatic1** [comment]
@diamondhands0 Thanks for this thoughtful response. I do think it's great that there is a passionate community forming and as a founder myself, I completely understand and appreciate how hectic things must be and how much effort it has taken to build something like this. I think you should be extremely proud that what you've built has reached the point where the community is stepping up and wanting more of a say, and I know you are working hard to hand over the reigns as soon as possible.

I think in general most people will agree that a move to POS is the right move from a technical perspective. However, I think there definitely has been a misunderstanding over the genesis block and the pre-mine. It would be valuable to share that information more broadly so that people are clear that the 6M tokens in the genesis block were purchased in the exact same was as the 6M tokens in the following blocks. I know I've had multiple discussions in the Discord community explaining this but still very few people are aware of what actually happened. I think if more people knew this it would help resolve concerns about moving to POS since the assumption by many people is that BitClout is not well distributed enough.

The last thing I'll say is that it's easy point out mistakes from the sidelines, but we should all remember how much blood, sweat, and tears must have gone into building this protocol. Imagine working for years building something with no indication that it's going to work at all and then launching it to the world and having it blow up so quickly (in addition to dealing with all kinds of controversy). It's impossible to do things perfectly, but I'm impressed by how the founding devs have reacted and handled everything so far and am very excited to see the dev community grow to include anyone and everyone capable and interested in contributing.

**Comment by FreeTrade** [comment]
>On MEV and 51% attacks: BitClout is resistant to these attacks at present due to the "trusted block producer" setup. I'm not going to go into it here, as this deserves a post on its own, but you can trace the --trusted_block_producer flag and read the comments if you're curious about how it works.

This certainly does deserve a post on its own! I'm very interested to hear if there are plans to decentralize mining.

**Comment by artzdata** [comment]
Thanks @diamondhands0 for the thoughts and updates. Appreciate you and the tireless work you and team are putting in to bring this to fruition.

**Comment by bitcloutrugpull** [comment]
This entire idea that PoS is more efficient and that Pow is bad is ridiculous and is a narrative that plays to those who already have coins.  Miners seek out the cheapest power rates they can possibly get.  That means they go to areas that have excess power because of supply and demand metrics and the influence on price.  Miners seek renewable energy sources because it's good for their bottom line.  

Miners who mine coins they like, normally do so because it is cheaper for them to acquire coins via mining than by buying outright.  Of course there are those who will dump, but assuming that 100% of the mining emission would be dumped is ludicrous.  100k coins a year at the old emission with maybe even a high 75 percent of them being dumped is only 75k coins to be absorbed by an ecosystem that already has 11 million in circulation.  Do the math, that is 0.006% of the circulating supply.  That's really going to hurt the Bitclout value metrics.  <roll eyes>  Current valuation is 2.2 billion at the same $200 coin price point used for the 20 million in emissions.  That's less than 1% of the market cap.  If that hurts you, you are destined to fail anyways.  The points given to rationalize this are retarded.

I don't know any miner that mines a coin where they are only taking 5% in profit for every coin they mine.  That is suicide for a miner.  All the reasons and numbers to attempt to justify the action are just made up to sustain the narrative even though it isn't backed by any real metrics and are numbers just shot from the hip.

There is more to this story.  Much more. The plebes are being fed a load because there is no real logic to the argument as laid out above.

**Comment by donhardman** [comment]
Well, first thanks for the answer. 

But I have a kinda offtopic question. 

Can you clarify about the swap identity request also here that allows admins to change any username/public key to another one? And looks like that means get all control, balances, steal usernames, and so on.

Am I right?

That's really much power. Will you give this possibility to people or remove it from the node?

**Comment by jazzyjackson** [comment]
> All of us hold BitClout. We are all aligned by the fact that we all own the same asset. 

@diamondhands0  if you wanted  #me to believe that our incentives are aligned you would have sent the company Bitcoin to a burn address.

**Comment by JRedinger** [comment]
I listen for things. I pay attention for things. I try to be very careful with my trust, like many of us, like you I'm sure, we've experienced the pain of trusting the wrong people in crypto. 

@diamondhands0, I trust you.

Please take adequate time to consider as many angles as you can for the POS foundation you are implementing. This is your opportunity to encode the balance of influence and incentive that will serve as the permanent foundation for your vision for this platform. As much as possible, please ensure that the interests of everyone you are trying to help through your creation is well cared for permanently as a fundamental feature of the consensus and governance protocols you and the rest of the team have designed and chose to implement here.

Thank you so much for all your hard work. I'm with you.

**Comment by Elrickerikose** [comment]
Waking up and entering clubhouse with strange discussion was...strange; now, after reading this thread I at least understand part of the reason why other people are not happy. Besides having BS in IT/BM , my knowledge of blockchain  and crypto are just starting. I didn't participate in mining, so the change didn't impact me directly, but as a BitClout enthusiast, who devotes almost 24/7  to the platform, i would be more happy, if any decisions would be properly discussed by community, before implementing.

**Comment by bloated-devish** [comment]
> This entire idea that PoS is more efficient and that Pow is bad is ridiculous and is a narrative that plays to those who already have coins. Miners seek out the cheapest power rates they can possibly get. That means they go to areas that have excess power because of supply and demand metrics and the influence on price. Miners seek renewable energy sources because it's good for their bottom line.

Seeking excess power and renewable energy is a lie and impossible for anyone mining as individual or even in GPU pools. It is only possible for large companies that can cut a deal with the local grid. These are typically ASIC mining companies.

Most of the individual miners and pools you are talking about will anyway evaporate in 2 months even if PoW wasn't changed - when all the FPGA folks enter the game. When they do, there will be nothing left for us regulars to mine and at some point they'll also get outgamed by ASIC folks.

In the end, mining will simply be done by large ASIC mining companies. On the other hand, as long as you have minimum stake, you can start earning rewards. Even that isn't often needed - most exchanges allow you to start staking and earning rewards even if you don't meet the minimum threshold. In my opinion, for small folks like me, it is better if it becomes PoS, as I can indeed earn money even if I don't meet the minimum threshold, thanks to crypto exchanges that will pool money for validation nodes.

Side note on ETH: ETH does have a profitable mining rate today - and that's mostly because of the gas fee. Even Mr. Buterin agrees long term PoS is the way to go and believes for the chain's long term sustenance, these astronomical gas fees are NOT good.

> 
> Miners who mine coins they like, normally do so because it is cheaper for them to acquire coins via mining than by buying outright. Of course there are those who will dump, but assuming that 100% of the mining emission would be dumped is ludicrous. 100k coins a year at the old emission with maybe even a high 75 percent of them being dumped is only 75k coins to be absorbed by an ecosystem that already has 11 million in circulation. Do the math, that is 0.006% of the circulating supply. That's really going to hurt the Bitclout value metrics. (roll eyes) Current valuation is 2.2 billion at the same $200 coin price point used for the 20 million in emissions. That's less than 1% of the market cap. If that hurts you, you are destined to fail anyways. The points given to rationalize this are retarded.

^ This is again - going to go away for GPU miners like you and me. We can't compete on economies of scale that FPGA and ASIC miners will bring to the table. Why aren't they in today? There is cost of development - and the lead time is all we got before we become unprofitable.

> 
> I don't know any miner that mines a coin where they are only taking 5% in profit for every coin they mine. That is suicide for a miner. All the reasons and numbers to attempt to justify the action are just made up to sustain the narrative even though it isn't backed by any real metrics and are numbers just shot from the hip.
> 
> There is more to this story. Much more. The plebes are being fed a load because there is no real logic to the argument as laid out above.

I am 100% with you that the change should have been explained better and communicated in a more orderly fashion. But from what you are championing, it looks like you are not standing for the plebs like you and me, but instead for the large ASIC and FPGA mining companies.

**Comment by ViktoriiaSergiienko** [comment]
> Прокидатися і входити в клуб з дивною дискусією було ... дивним; тепер, прочитавши цю тему, я принаймні розумію частину причини, через яку інші люди не задоволені. Окрім того, що я маю ступінь бакалавра в галузі ІТ / BM, мої знання про блокчейн та криптовалюту тільки починаються. Я не брав участі в майнінгу, тому зміна не вплинуло на мене безпосередньо, але як ентузіаст BitClout, який присвячує платформу майже цілодобово та без вихідних, я був би більш щасливим, якщо б якісь рішення були належним чином обговорені спільнотою, перед впровадженням.

completely agree

**Comment by sungkhum** [comment]
Yikes

**Pull request merged**
